ra-optimisation/rao-api/src/main/java/com/farao_community/farao/rao_api/RaoParameters.java

```diff
@@ -50,6 +55,8 @@ public static RaoParameters load(PlatformConfig platformConfig) {
         Objects.requireNonNull(platformConfig);
 
         RaoParameters parameters = new RaoParameters();
+        //parameters.setDcMode(config.getBooleanProperty("dc-mode", DEFAULT_DC_MODE));
+        //parameters.setAcToDcFallback(config.getBooleanProperty("ac-to-dc-fallback", DEFAULT_AC_TO_DC_FALLBACK));
```

Comment:
@baptsegu @TheMaskedTurtle 
Do you know if I have to write such a thing here? (ie get the booleans from platformConfig)

Reply:
I would say yes otherwise you can never load the config. But I don't know how to test that properly.